### DISCUSSION TITLE
Backup for database structure without data

### DISCUSSION THREAD

**Discussion by MuhammedMukdad** [spatie/laravel-backup#1592] [Q&A]
Can I take a backup for my mysql database structure without the data ?

**Comment by angeljqv** [comment]
Read documentation

**Reply by MuhammedMukdad** [reply]
I didn't find anything to help me in the documentation.
Can you share the link that can help me? , please.

**Comment by erikn69** [comment]
You can set dumper configs for anything, read documentation about adding extra custom options(`--no-data`)
https://github.com/spatie/laravel-backup/blob/a0af6e31ea0f541505ebe3f37cb4d559bfcc950f/config/backup.php#L80
https://github.com/spatie/db-dumper
https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html
